pages/api/login.js

```diff
@@ -28,6 +28,7 @@ async function loginHandler(req, res) {
       lastName,
       role,
       defaultEntityType,
+      homepage: `/admin/content-manager/${defaultEntityType}`
```

Comment:
The default homepage should be `/admin` 

```suggestion
      homepage: `/admin`
```